Source/Core/DolphinQt/Config/GameConfigWidget.cpp

```diff
@@ -90,6 +90,7 @@ void GameConfigWidget::CreateWidgets()
   m_sync_gpu = new QCheckBox(tr("Synchronize GPU thread"));
   m_enable_fast_disc = new QCheckBox(tr("Emulate Disc Speed"));
```

Comment:
This isn't strictly your change, but since you've got nicer handling of reversed options, it might be clearer to rename `m_enable_fast_disc` to `m_emulate_disc_speed` so that it's more obvious that that one needs to be reversed as well.